README.md

```diff
@@ -83,7 +83,7 @@ RICK | [komodo](https://github.com/komodoplatform/komodo) | 0.5.0 | [![dPOW Stat
 SEC | [komodo](https://github.com/komodoplatform/komodo) | 0.5.0 | [![dPOW Status](http://159.69.45.70/svg/SEC_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
 SUPERNET | [komodo](https://github.com/komodoplatform/komodo) | 0.5.0 | [![dPOW Status](http://159.69.45.70/svg/SUPERNET_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
 THC | [komodo](https://github.com/komodoplatform/komodo) | 0.5.0 | [![dPOW Status](http://159.69.45.70/svg/THC_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
-VRSC | [verus](https://github.com/VerusCoin/VerusCoin) | 0.6.0-7 | [![dPOW Status](http://159.69.45.70/svg/VRSC_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
+VRSC | [verus](https://github.com/VerusCoin/VerusCoin) | [0.6.0-7 view instructions](https://gist.github.com/imylomylo/19465a6dc60c1cafdde5174844e2cd4a) | [![dPOW Status](http://159.69.45.70/svg/VRSC_badge.svg?maxAge=60)](https://komodostats.com) | dPoW-mainnet
```

Comment:
can you plz just link the 0.6.0-7 in the version field to https://github.com/VerusCoin/VerusCoin/releases/tag/v0.6.0-7 (i.e. removing the gist link - we assume that naming repo, branch and version must be sufficient). We will add a `docs` folder to the dPoW codebase where we could add more detailed instructions if that is required.

Reply:
Closing this PR, submitting a new one with only the requested change for neater history (Doing it from browser).